### PR TITLE
Ts 1225

### PR DESCRIPTION
**Pull request by Nikita-Smirnov-Exactpro**
This version should be released ASAP

### CODE REVIEW

**Review by Xanclry — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,37 +1,27 @@
...
 This microservice allows sending and receiving messages via FIX protocol
 
 ## Configuration
 
-+ *autoStart* - enables/disable auto-starting of session on box start (`true` by default)
-+ *autoStopAfter* - time in seconds after which session will be automatically stopped (`0` by default = disabled)
++ *sessions* - list of session settings
 + *maxBatchSize* - max size of outgoing message batch (`1000` by default)
 + *maxFlushTime* - max message batch flush time (`1000` by default)
++ *batchByGroup* - batch messages by group instead of session alias and direction (`true` by default)
```
Comment (ivandruzhinin) [inline README.md:10]: Mentions of groups should be removed until migration to core v3

File: README.md [hunk 2]
```diff
@@ -222,10 +224,9 @@ spec:
   image-version: 0.0.1
   type: th2-conn
   custom-config:
-    autoStart: true
-    autoStopAfter: 0
-    maxBatchSize: 100
+    maxBatchSize: 1000
     maxFlushTime: 1000
+    batchByGroup: false
```
Comment (ivandruzhinin) [inline README.md:229]: Same here

File: README.md [hunk 3]
```diff
@@ -283,17 +284,11 @@ spec:
...
-        - publish
-        - raw
-    - name: incoming_messages
+    - name: outgoing
       connection-type: mq
       attributes:
-        - first
         - publish
+        - store
         - raw
```
Comment (ivandruzhinin) [inline README.md:292]: Should replaced with 2 pins split by direction (see example in core v2 readme)

File: build.gradle [hunk 4]
```diff
@@ -36,18 +44,20 @@ repositories {
...
+    api platform('com.exactpro.th2:bom:4.1.0')
 
-    implementation 'com.exactpro.th2:conn-dirty-tcp-core:1.0.0'
-    implementation 'com.exactpro.th2:common:3.31.1'
-//    implementation 'com.exactpro.th2:common:3.42.1-TH2-2212-common-cannot-recover-channel-level-exceptions-3540133251-SNAPSHOT'
+    implementation 'com.exactpro.th2:common:3.44.0'
+    implementation 'com.exactpro.th2:netty-bytebuf-utils:0.0.1'
+    implementation ('com.exactpro.th2:conn-dirty-tcp-core:2.0.5') {
+        exclude group: 'org.slf4j', module: 'slf4j-log4j12'
+        because 'Projects should use only slf4j-api, without coupling to a sertain implementation'
```
Comment (ivandruzhinin) [inline build.gradle:53]:
```suggestion
        because 'Projects should use only slf4j-api, without coupling to a certain implementation'
```

**Review by ivandruzhinin — CHANGES_REQUESTED** [review]

**Review by ivandruzhinin — APPROVED** [review]

**Pull request merged**